lib/models/lookup.js

```diff
@@ -34,13 +34,17 @@ function LookupModelFactory (Model, assert, Errors, Promise, Constants) {
             }
         },
         findByTerm: function (term) {
-            return this.find({
-                or: [
-                    { node: term },
-                    { macAddress: term },
-                    { ipAddress: term }
-                ]
-            });
+            var query = {}; //empty query will find all entries
```

Comment:
if ```term``` is empty, ```findByTerm``` will find nothing sounds reasonable. 
The better place to add this logic maybe in on-http/lib/api/lookups.js. 
When ```req.query.q``` is empty, call ```find``` directly.